x/bank/spec/README.md

```diff
@@ -1,13 +1,26 @@
-<!--
-order: 0
-title: Bank Overview
-parent:
-  title: "bank"
--->
```

Comment:
Please add this back, this is needed for the docs site. I commented on the merged commit, moving the comment here